Packs/CrowdStrikeFalconX/Playbooks/CrowdStrike_Falcon_X_-_Detonate_URL.yml

```diff
@@ -0,0 +1,378 @@
+id: FalconX - Detonate URL
+version: -1
+name: FalconX - Detonate URL
```

Comment:
@EliorKedar do we want to name it like this?
or `CrowdStrike Falcon X - Detonate URL`?
same for the detonate file.

Reply:
@bakatzir CrowdStrike Falcon X - Detonate URL